src/components/SideBar/style/sideBar.scss

```
@@ -35,6 +36,7 @@
         left: 80px;
         width: 110px;
         background-color: $gray;
+        color: white;
```

use variable

---

thx, you right about it.
```suggestion
        color: $white;
```